gorm/collection_operators.go

```diff
@@ -62,6 +67,40 @@ func ApplyCollectionOperatorsEx(ctx context.Context, db *gorm.DB, obj interface{
 	return db, nil
 }
 
+func ApplyCollectionOperatorsWithSearchingEx(ctx context.Context, db *gorm.DB, obj interface{}, c CollectionOperatorsConverter, f *query.Filtering, s *query.Sorting, p *query.Pagination, fs *query.FieldSelection, sc *query.Searching, fieldsForFTS []string) (*gorm.DB, error) {
```

Comment:
Because this is a new operator with a separate function, it won't be handled in the normal protoc-gen-gorm generated handlers without changes there.